app.py

```diff
@@ -32,6 +35,12 @@
 
 app = Flask(__name__)
 
+log_handler = CloudWatchLogHandler(
+    log_group_name=f"citybase_{flask_env}", log_stream_name="postback_stream"
+)
+logging.basicConfig(level=logging.INFO)
+logging.getLogger().addHandler(log_handler)
```

Comment:
it's helpful for debugging the `watchtower` functionality, but i wonder if we might want to skip the cloudwatch handler when doing local dev. do we want to create crufty log groups when running locally?

Reply:
I think we should set the log-group to only retain so many days of logs, but otherwise, I'm pro keeping the dev logging in cloudwatch. I think it's helpful to be able to search, expand, collapse, filter the logs in cloudwatch, especially during the development process, and the quality of our logging will be improved if we log into the same mechanism we use in staging and production. 

I hear you about not creating cruft, and I think we could do ourselves a huge leg-up if we were to assign an issue for someone to get into our cloudwatch console and clean up all the 5+ years of cruft we have in there -- I think we'd clean up like 90% of the entries and reduce the need to limit our creation of log groups.

Reply:
@johnclary something that I will add to the README (thank you for reminding me to update that) is to note that the logs in cloudwatch for local dev are only retained for a short time, so the cruft gets swept away